### DISCUSSION TITLE
[Feature] Info display of a picture should show the Albums it is in

### DISCUSSION THREAD

**Discussion by Frava735** [immich-app/immich#11544] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When clicking on the 'i' for any picture,  it shows a lot of information,  but never the album thumbnail it is present in.
Multiple album thumbnails should be displayed when picture is part of multiple albums (same goes for the faces... multiple faces, multiple thumbnails).

These album thumbnails would be a link to the album it represents. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is already supported

![image](https://github.com/user-attachments/assets/9f1f64ef-d59c-43a2-858a-4d7d2a2f16bc)

**Reply by Frava735** [reply]
It is displayed in Web app.
Not in mobile app.

I shouldn't have checked the web tick box.
So the feature request is still true for mobile app.
